src/ui/component/fileProperties/view.jsx

```diff
@@ -0,0 +1,43 @@
+// @flow
+import * as icons from 'constants/icons';
+import * as React from 'react';
+import { parseURI } from 'lbry-redux';
+import Icon from 'component/common/icon';
+import FilePrice from 'component/filePrice';
+
+type Props = {
+  uri: string,
+  downloaded: boolean,
+  claimIsMine: boolean,
+  isSubscribed: boolean,
+  isNew: boolean,
+  rewardedContentClaimIds: Array<string>,
+  tags: Array<string>,
+};
+
+function FileProperties(props: Props) {
+  const { uri, downloaded, claimIsMine, rewardedContentClaimIds, isSubscribed, isNew, tags } = props;
+  const { claimId } = parseURI(uri);
+  const isRewardContent = rewardedContentClaimIds.includes(claimId);
+
+  return (
+    <React.Fragment>
+      <div className="file-properties">
+        {isSubscribed && <Icon icon={icons.SUBSCRIPTION} />}
+        {!claimIsMine && downloaded && <Icon icon={icons.DOWNLOAD} />}
+        {isRewardContent && <Icon iconColor="red" icon={icons.FEATURED} />}
+        {isNew && <span className="badge badge--alert">{__('NEW')}</span>}
+        <FilePrice hideFree uri={uri} />
+      </div>
+      <div className="file-properties">
+        {tags.slice(0, 3).map(tag => (
```

Comment:
What about privileging tags I follow? Also, the slice should be sorted or otherwise consistently ordered (one claim may have tags `apple, banana, orange`, while the other has `orange, banana, apple`).